api/src/pcapi/alembic/versions/20240403T121142_251c72cb3044_add_validationauthor_column_for_booking_table.py

```diff
@@ -0,0 +1,27 @@
+"""
+add validationAuthorType column for Booking table
+"""
+
+from alembic import op
+import sqlalchemy as sa
+from sqlalchemy.dialects import postgresql
+
+from pcapi.core.bookings.models import BookingValidationAuthorType
+
+
+# pre/post deployment: pre
+# revision identifiers, used by Alembic.
+revision = "251c72cb3044"
+down_revision = "1c0c3459615b"
+branch_labels: tuple[str] | None = None
+depends_on: list[str] | None = None
+
+
+def upgrade() -> None:
+    booking_author_type = postgresql.ENUM(BookingValidationAuthorType, name="validationAuthorType")
+    booking_author_type.create(op.get_bind(), checkfirst=True)
+    op.add_column("booking", sa.Column("validationAuthorType", booking_author_type, nullable=True))
```

Comment:
il faut trouver le moyen de rajouter le "IF NOT EXISTS" et créer la colonne manuellement avant la MES/MEP sinon on va avoir des problemes pour la MEP.

Reply:
c'est plus prendre le lock sur la table qui me fait peur, la création de la colonne sera tres rapide

Reply:
pcapi-staging> ALTER TABLE booking
 ADD "validationAuthorType" VARCHAR(20) NULL;
You're about to run a destructive command.
Do you want to proceed? [y/N]: y
Your call!
ALTER TABLE
Time: 0.008s
pcapi-staging>